IFC4x3/Properties/n/NominalDepth_1shYs0qXWHuO00025QrE$V/DocProperty.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NominalDepth_1shYs0qXWHuO00025QrEV" Name="NominalDepth" UniqueId="1shYs0qXWHuO00025QrE$V" PrimaryDataType="IfcPositiveLengthMeasure">
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="NominalDepth" UniqueId="1shYs0qXWHuO00025QrE$V" PrimaryDataType="IfcPositiveLengthMeasure">
 	<Localization>
 		<DocLocalization Locale="en" Name="Nominal Depth" />
 		<DocLocalization Locale="ja-JP" Name="深さ">
```

Comment:
Can we transfer the localization to the harmonized keep element NominalDepth_0HZnE0qUaHuO00025QrEV

Reply:
Yes, I have copied the localization in subsequent commit.